apps/site/pages/docs/global-tokens/Colors.mdx

```diff
@@ -0,0 +1,299 @@
+import {
+  TokenColors,
+  TokenColor,
+  TokenTable,
+  TokenRow,
+  TokenDivider,
+} from 'site/components/Tokens'
+
+<header>
+
+# Colors
```

Comment:
Shouldn't this file be named as `colors.mdx`? (all small caps)

Reply:
oh, sure. I thought I've done that. 🙈